platforms/Windows/python/python.wxi

```diff
@@ -0,0 +1,154 @@
+        <File Source="$(PythonRoot)\libffi-7.dll" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\libssl-1_1$(ArchSuffix).dll" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\pyexpat.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python.cat" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python.exe" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python3.dll" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python310.dll" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python310.zip" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\python310._pth" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\pythonw.exe" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\select.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\sqlite3.dll" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\unicodedata.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\winsound.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_asyncio.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_bz2.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_ctypes.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_decimal.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_elementtree.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_hashlib.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_lzma.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_msi.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_multiprocessing.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_overlapped.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_queue.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_socket.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_sqlite3.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_ssl.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_uuid.pyd" />
+      </Component>
+
+      <Component>
+        <File Source="$(PythonRoot)\_zoneinfo.pyd" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="EmbeddedPythonLicense">
+      <Component Directory="toolchain_$(VariantName)_usr_share_licenses">
```

Comment:
I don't think that we should be adding this to the toolchain layout but rather the python subdirectory.